pkg/cli/admin/release/info.go

```diff
@@ -775,6 +775,33 @@ func (o *InfoOptions) LoadReleaseInfo(image string, retrieveImages bool) (*Relea
 				errs = append(errs, err)
 				return true, nil
 			}
+			// Try the registry/repo passed from the user first.  If image does not exist,
+			// proceed with the release info from the release image-references, from readReleaseImageReference above
+			for _, tag := range is.Spec.Tags {
+				// this is user-passed image name
+				userGivenImage, err := imagereference.Parse(image)
+				if err != nil {
+					errs = append(errs, err)
+					return true, nil
+				}
```

Comment:
This could be outside of the loop.

Reply:
Actually looking inside `ParseReference` invoked at the beginning of the method that one already has that invocation see https://github.com/openshift/oc/blob/29e9a330f0ab0eed17b5ba6a8cc198ccc4332567/pkg/cli/image/imagesource/reference.go#L96 Iow. you could just check `ref.Type == DestinationRegistry` and drop this above code entirely.

Reply:
thanks, i've updated.  much cleaner. 
userGivenImage.AsRepository() == `ref.Ref.AsRepository()` :) so i can drop the userGivenImage nonsense altogether